components/cam/src/control/cam_history.F90

```diff
@@ -2379,6 +2376,7 @@ subroutine fldlst ()
     deallocate(gridsontape)
 
     call print_active_fldlst()
+
```

Comment:
Does this file match the E3SM version? Or were these just manual formatting changes?